DAQ/src/ArtBinaryPacketsFromDigis_module.cc

```diff
@@ -1114,6 +1116,15 @@ uint8_t ArtBinaryPacketsFromDigis::compressCrvDigi(int adc) {
     toReturn = 255;
   return toReturn;
 }
+*/
```

Comment:
I must say I'm not a fan of leaving commented code around. If it's no longer useful, delete it.

Reply:
This is code that was supposed to be used for the compression algorithm. However, Simon's data doesn't use compression, yet. I can delete this code section until the compression algorithm is implemented.